### ISSUE TITLE
[Opposing Force] Glock reload bug and a feature request

### ISSUE THREAD

**Issue by BlackShadow306** [ValveSoftware/halflife#2453]
Opposing force still having the old infinite glock reload issue.

Also is it possible to implement updated scoreboard to Opposing Force? It's still using the Old WON Scoreboard.

**Comment by StevenKal** [comment]
Also (out of context), if the "colored menus" support could be added too. I've not tested on the beta, but on a build #7561 they aren't available.
This feature (menus in color like under CS, CZ, DoD) has been added from the 2013 updates on your main other game mods as HL, TFC, DMC, but not on HLOF, neither Ricochet too from that I remember.
Probably some other "common" features were not added on these both games, did they smell bad?!

So it will be nice to have that as soon as possible.

**Comment by schmurtzm** [comment]
yes , not really hard to update, we could replace it by the HUD of HL1 but Opposing HUD include capture the flag features that HL1 HUD does not have.

This old HUD is a problem (I mean this is not just a feature request but a real bug)  : when you have more than 100 death it is not displayed correctly : the hundreds don't show up.

An update would be welcome !

**Comment by barney-heatwave** [comment]
Gonna bump this. Seeing an updated scoreboard would be very nice, including a CVar in all Half-Life Engine games like `cl_oldscoreboard` to toggle it

**Comment by djearthquake** [comment]
This is not going to happen.  GoldSrc is out of production and it would be too expensive.
